### DISCUSSION TITLE
Markdown text variable

### DISCUSSION THREAD

**Discussion by adpadi** [quarto-dev/quarto-cli#1048] [Q&A] [answered]
I am looking for a way to use a variable within markdown text using Jupyter Lab. For instance

name = "Jennifer"
number = 412.3

And resulting markdown text is: Jennifer gave an answer of 412.3.

I've found some glimmers of a solution around, but nothing has worked for me yet. Seems like the Jupyter Lab/Notebook difference is causing problems. Any suggestions?

**Comment by jjallaire** [comment, marked as answer]
Here is how you do it in Quarto documents: https://quarto.org/docs/computations/execution-options.html#jupyter

This will also work in Jupyter Lab but is not exactly the user experience I'm sure you want. This is being discussed in the Jupyter community but hasn't yet reached a resolution: https://discourse.jupyter.org/t/inline-variable-insertion-in-markdown/10525

**Reply by adpadi** [reply]
The first link appears to be working for me in Jupyter Lab now. I had tried before and must have had something wrong. I just replaced "radius" with my variable (and removed the ````{python}``` `wrap if not obvious).

I'll keep an eye out for the second solution proposed. 
Thx
